src/Farmer/Arm/Databricks.fs

```diff
@@ -0,0 +1,65 @@
+[<AutoOpen>]
+module Farmer.Arm.Databricks
+
+open Farmer
+open Farmer.Databricks
+open Farmer.Arm.Network
+open System
+
+let workspaces = ResourceType ("Microsoft.Databricks/workspaces", "2018-04-01")
+
+type ByovConfig =
+    { Vnet: ResourceRef<ByovConfig>
+      PublicSubnet: ResourceName
+      PrivateSubnet: ResourceName }
+
+type EncryptionConfig =
+    { KeySource: string
+      KeyName: string
+      KeyVersion: string
+      KeyVault: ResourceName }
+
+type Workspace =
+    { Name: ResourceName
+      Location: Location
+      ManagedResourceGroupId: ResourceName
+      Sku: Sku
+      EnablePublicIp: FeatureFlag
+      PrepareEncryption: FeatureFlag
+      Encryption: EncryptionConfig option
+      ByovConfig: ByovConfig option
+      Tags: Map<string,string> }
+    interface IArmResource with
+        member this.ResourceId = workspaces.resourceId this.Name
+        member this.JsonModel =
+            {| workspaces.Create(this.Name, this.Location, tags = this.Tags) with
+                sku =
+                    {| name =
+                        match this.Sku with
+                        | StandardTier -> "standard"
+                        | PremiumTier -> "premium" |}
+                properties =
+                    {| managedResourceGroupId =
+                        let expr = sprintf "concat(subscription().id, '/resourceGroups/', '%s')" this.ManagedResourceGroupId.Value
+                        ArmExpression.create(expr).Eval()
+                       parameters = Map [
+                        "enableNoPublicIp", box this.EnablePublicIp.AsBoolean
```

Comment:
I'm not sure that this will work as intended. Enabling this option actually disables the use of public IPs so the boolean value of EnablePublicIp should be negated.

Reply:
Doh. Great spot!

Reply:
Sorry about that.

Reply:
No worries. I blame Microsoft's naming for the confusion